docs/ruleset_functions/is_cooling_type_DX(hvac_b.id).md

```diff
@@ -0,0 +1,23 @@
+# is_cooling_type_DX  
+
+**Description:** Returns TRUE if the HVAC system has DX cooling. Returns FALSE if the HVAC system has anything other than DX cooling or if it has more than 1 cooling system.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled with DX cooling.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_cooling_type_DX**: Returns TRUE if the HVAC system has DX cooling AND only one cooling system . Returns FALSE if the HVAC system has a cooling system type other than DX or if it has more than one cooling system.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Check that there is only one cooling system associated with the HVAC system: `if Len(hvac_b.cooling_system) != 1: is_cooling_type_DX = FALSE`  
```

Comment:
Suggest to make this a separate sub-function as it is repeated in all other sub-functions.